docs/users/oauth_server.md

```diff
@@ -18,26 +18,12 @@ You can install it with the following command:
 composer require ibexa/oauth2-server --with-all-dependencies
 ```
 
-Then, in `config/bundles.php`, at the end of an array with a list of bundles, add the following two lines :
-
-```php
-<?php
-
-return [
-    // A lot of bundles…
-    Ibexa\Bundle\OAuth2Server\IbexaOAuth2ServerBundle::class => ['all' => true],
-    League\Bundle\OAuth2ServerBundle\LeagueOAuth2ServerBundle::class => ['all' => true],
-];
-```
-
 Add the tables needed by the bundle:
 
 === "MySQL"
 
     ```bash
```

Comment:
````suggestion
=== "MySQL"

    Be sure to have `utf8mb4` as default charset and `utf8mb4_unicode_520_ci` as default collation.

    ```bash
````

Reply:
Still missing an how-to-do-it…

Reply:
Charset/Collation issue moved to [IBX-8855](https://issues.ibexa.co/browse/IBX-8855).

This PR shouldn't focus on this anymore.